### ISSUE TITLE
Splunk logs incorrect container.image name & tag

### ISSUE THREAD

**Issue by dmanojbaba** [signalfx/splunk-otel-collector-chart#699]
Hello, This issue is related to the PR https://github.com/signalfx/splunk-otel-collector-chart/pull/285

We recently migrated from `Splunk Connect for Kubernetes` to `Splunk OpenTelemetry Collector for Kubernetes`

We are using jfrog artifactory registry ports method for all our container images:
```
Example: dockerhub
artifactory_domain_name:port1/splunk/fluentd-hec:tag

Example: quay.io
artifactory_domain_name:port2/signalfx/splunk-otel-collector:tag
```
With Splunk connect for k8s, container logs the `container_image` field had the correct value:
```
container_image = artifactory_domain_name:port1/my-app/my-image:my-tag
```
However, in otel collector, it now as two new fields with incorrect values:
```
container.image.name = artifactory_domain_name
container.image.tag = port1/my-app/my-image
```
It should have been like this:
```
container.image.name = artifactory_domain_name:port1/my-app/my-image
container.image.tag = my-tag
```

The problem is, we are unable to concatenate these two fields to get the required value because the actual tag is missing in the container.image.tag (due to incorrect regex?)

Could someone please help?

**Comment by dmanojbaba** [comment]
Thanks @jvoravong
I had a chat with @mattymo in splunk-usergroups Slack #opentelemetry channel yesterday.
We found the root cause and I have created an [issue](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/20239 ) for the same.

https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/20239

**Comment by dmitryax** [comment]
The bug is fixed upstream and will be available as part of v0.75.0 release
